modules/geo-layers/src/tile-layer/tileset-2d.js

```diff
@@ -24,7 +25,7 @@ const TILE_STATE_VISIBLE = 1;
 const TILE_STATE_PLACEHOLDER = 3;
 const TILE_STATE_HIDDEN = 4;
 // tiles that should be displayed in the current viewport
-const TILE_STATE_SELECTED = 5;
+export const TILE_STATE_SELECTED = 5;
```

Comment:
This was intended to be a private flag. Is the `isVisible` field not sufficient?

Reply:
`isVisible` is computed as `tile.state & TILE_STATE_VISIBLE`, so `isVisible` will be true for any of `TILE_STATE_VISIBLE`, `TILE_STATE_PLACEHOLDER`, or `TILE_STATE_SELECTED`.

But for cancelling requests, I think we want to test only for `TILE_STATE_SELECTED`. I.e. we should display a placeholder tile _if it already exists_, but not wait for placeholder tiles to load before loading selected tiles.

I originally used `isVisible`, but it means that if you zoom in, all parent tiles are still loaded, because all parent tiles are continuously visible, but not selected.

I could add an `isSelected` prop to the tile?

Reply:
That works too.